package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node server.js",
+    "start": "nodemon server.js",
```

Comment:
`nodemon` is convenient for development, but make sure that your start command on render is not `npm start` because we don't want to use `nodemon` in a deployed application. On render you would want the start command to be `node server.js`.

Alternatively, you could also create a separate start script for development:
```
"scripts": {
  "dev": "nodemon server.js",
  "start": "node server.js", 
  ...
}
```